koji_containerbuild/plugins/cli_containerbuild.py

```diff
@@ -152,8 +157,64 @@ def parse_arguments(options, args, flatpak):
     return build_opts, args, opts, parser
 
 
-def handle_build(options, session, args, flatpak):
-    build_opts, args, opts, parser = parse_arguments(options, args, flatpak)
+def parse_source_arguments(options, args):
+    "Build a source container"
+    usage = _("usage: %prog source-container-build [options] target")
+    usage += _("\n(Specify the --help global option for a list of other help "
+               "options)")
+    parser = OptionParser(usage=usage)
+    parser.add_option("--scratch", action="store_true",
+                      help=_("Perform a scratch build"))
+    parser.add_option("--wait", action="store_true",
+                      help=_("Wait on the build, even if running in the "
+                             "background"))
+    parser.add_option("--nowait", action="store_false", dest="wait",
+                      help=_("Don't wait on build"))
+    parser.add_option("--quiet", action="store_true",
+                      help=_("Do not print the task information"),
+                      default=options.quiet)
+    parser.add_option("--background", action="store_true",
+                      help=_("Run the build at a lower priority"))
+    parser.add_option("--epoch",
+                      help=_("Specify container epoch. Requires koji admin "
+                             "permission."))
```

Comment:
Not needed. I'm not even sure if it's used for regular image builds.

Reply:
sure